### PR TITLE
Add BYO Certs Support To Clusters Module

### PR DESCRIPTION
**Pull request by git-ival**

### CODE REVIEW

File: clusters/main.tf [hunk 1]
```diff
@@ -23,35 +20,41 @@ terraform {
...
   name        = "${local.name}-${local.cluster_instance}-${count.index}"
   description = "TF imported cluster ${local.name}-${local.cluster_instance}-${count.index}"
+  labels      = var.cluster_labels
 }
 
 resource "aws_instance" "k3s-server" {
   ebs_optimized = true
   instance_type = var.server_instance_type
   ami           = data.aws_ami.ubuntu.id
+
```
Comment (rmweir) [inline clusters/main.tf:56]: Can this new line be removed?

File: clusters/main.tf [hunk 2]
```diff
@@ -3,15 +3,12 @@ terraform {
...
       source  = "rancher/rancher2"
-      version = "1.13.0"
     }
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.12.0"
     }
     template = {
       source  = "hashicorp/template"
-      version = "~> 2.2.0"
```
Comment (rmweir) [inline clusters/main.tf:14]: Why remove these?

File: clusters/variables.tf [hunk 3]
```diff
@@ -9,14 +9,15 @@ variable "cluster_count" {
   description = "Number of clusters to provision"
 }
 
-variable "ec2_instances_per_cluster" {
-  type        = number
-  description = "Number of EC2 instances per cluster"
```
Comment (rmweir) [inline clusters/variables.tf:14]: Why is this and `k3s_per_node` being removed?

**Review by rmweir — CHANGES_REQUESTED** [review]

File: clusters/main.tf (hunk 2 above)
Comment (git-ival) [reply clusters/main.tf:14]: We use the `clusters/.terraform.lock.hcl` file to lock in the provider versions so these were a bit redundant.

File: clusters/variables.tf (hunk 3 above)
Comment (git-ival) [reply clusters/variables.tf:14]: These vars aren't actually used in the `clusters` module.

**Review by rmweir — APPROVED** [review]
LGTM

**Pull request merged**